hummingbot/connector/exchange/ascend_ex/ascend_ex_exchange.py

```diff
@@ -338,8 +348,11 @@ async def _format_trading_rules(self, instruments_info: Dict[str, Any]) -> Dict[
         return trading_rules
 
     async def _update_account_data(self):
+        """
+        Updates account group and uid from data obtained from the exchange
+        """
         url = f"{CONSTANTS.REST_URL}/{CONSTANTS.INFO_PATH_URL}"
-        headers = self._ascend_ex_auth.get_headers(CONSTANTS.INFO_PATH_URL)
+        headers = self._ascend_ex_auth.get_auth_headers(CONSTANTS.INFO_PATH_URL)
```

Comment:
The auth headers should be added automatically by the RESTAssistant using the auth object configured in it